### ISSUE TITLE
A11y_Azure Data Studio Nov2023_Server_Server Properties Preview_Use of color: Required fields in "Server Properties Preview" dialog is not associated with "*"(Star).

### ISSUE THREAD

**Issue by Swati1700** [microsoft/azuredatastudio#25207]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #WCAG1.4.1; #Use of color; #FTP; #SH_Azure Data Studio_Nov2023_Linux_Dec2023; #SQL Azure Data Studio; #Linux;

### Environment Details:
Application Name: Azure Data Studio
OS Name: Ubuntu 22.04.2 LTS
Azure data studio 1.47.0

### Repro Steps:

1. Install Azure data studio application and open it
2. Start Orca screen reader.
3. Make sure to enable "Preview" feature in settings.
4. Tab till "Server" and press F10 key to open the context menu.
5. Tab till "Properties Preview" and press enter.
6. "Server Properties Preview" dialog will appear.
7. Tab till "Memory" tab and press enter.
8. You will observe that Orca will announce the fields as required but no "*"(Star) is associated with fields nor any note is provided to fields as it's required.

### Actual Result:
Required fields in "Server Properties Preview" dialog is not associated with "*"(Star). Neither any note is provided to fields as it's required.

Note: Similar issue is observed for required fields throughout the application.

### Expected Result:
Required fields in "Server Properties Preview" dialog should be associated with "*"(Star) or note needs to be provided to fields as it's required if all the fields in dialog are required.

### User Impact:
Sighted-users will not able to understand and identify the form fields as required or not if there is no any chevron is associated.

### Attachment:
[25207 (2).zip](https://github.com/microsoft/azuredatastudio/files/13952030/25207.2.zip)

**Comment by barbaravaldez** [comment]
Fixed by https://github.com/microsoft/azuredatastudio/pull/25243

**Comment by corivera** [comment]
@Swati1700 This issue has been fixed in the latest Ubuntu Insiders build of ADS [here](https://azuredatastudio-update.azurewebsites.net/latest/linux-deb-x64/stable). Can you verify that the issue is fixed?

**Comment by corivera** [comment]
@Swati1700 Can you verify that the issue is resolved?

**Comment by Swati1700** [comment]
Verified the issue in the environment-
Azure data studio 1.49.0 insider
The issue is fixed hence closing the bug. PFA.
[25207_Closed.webm](https://github.com/microsoft/azuredatastudio/assets/86710888/86272725-288c-4101-adb3-907d136bb785)
